bpf/http_types.h

```diff
@@ -175,12 +175,13 @@ const u8 ip4ip6_prefix[] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xff, 0xff};
 
 #ifdef BPF_DEBUG
 static __always_inline void dbg_print_http_connection_info(connection_info_t *info) {
-    bpf_dbg_printk("[http] s_h = %llx, s_l = %llx, d_h = %llx, d_l = %llx, s_port=%d, d_port=%d",
+    bpf_dbg_printk("[conn] s_h = %llx, s_l = %llx, s_port=%d",
```

Comment:
I split this so that we can use it on older kernels where bpf_printk allowed for at most 3 arguments.